orbit/pkg/luks/luks.go

```diff
@@ -0,0 +1,37 @@
+package luks
+
+import (
+	"github.com/fleetdm/fleet/v4/orbit/pkg/dialog"
+)
+
+type ErrCode int
+
+const (
+	ErrKeySlotFull       ErrCode = -22
+	ErrBadPassphrase     ErrCode = -1
+	entryDialogTitle             = "Enter disk encryption passphrase"
+	entryDialogText              = "Passphrase:"
+	retryEntryDialogText         = "Passphrase incorrect. Please try again."
+	infoFailedTitle              = "Encryption key escrow"
+	infoFailedText               = "Failed to escrow key. Please try again later."
+	infoSuccessTitle             = "Encryption key escrow"
+	infoSuccessText              = "Key escrowed successfully."
+	maxKeySlots                  = 8
+	randPassphraseLength         = 32
+)
+
+type KeyEscrower interface {
```

Comment:
@iansltx the implementation is currently stubbed

Reply:
@mostlikelee Do we need to push a client error here sometimes?